webpack.config.js

```diff
@@ -87,37 +90,56 @@ module.exports = config(({ development, production, version }) => ({
         transform: transformManifest('firefox')
       }
     ]),
-    production &&
-      new FileManagerPlugin({
-        onEnd: [
-          {
-            archive: [
-              {
-                source: 'dist/chrome',
-                destination: `dist/toggl-button-chrome-${version}.zip`
-              },
-              {
-                source: 'dist/firefox',
-                destination: `dist/toggl-button-firefox-${version}.zip`
-              }
-            ]
-          },
-          {
-            delete: ['dist/chrome', 'dist/firefox']
-          }
-        ]
+    new FileManagerPlugin({
+      onEnd: [
+        {
+          copy: [
+            { source: 'dist/scripts/**/*', destination: 'dist/chrome/scripts' },
+            { source: 'dist/scripts/**/*', destination: 'dist/firefox/scripts' }
+          ]
+        },
+        production && {
+          delete: [
+            'dist/**/*.js.map'
+          ],
+          archive: [
+            {
+              source: 'dist/chrome',
+              destination: `dist/toggl-button-chrome-${version}.zip`
+            },
+            {
+              source: 'dist/firefox',
+              destination: `dist/toggl-button-firefox-${version}.zip`
+            }
+          ]
+        }
+      ]
+    }),
+    production && release &&
+      new BugsnagSourceMapUploaderPlugin({
+        apiKey: process.env.BUGSNAG_API_KEY,
```

Comment:
This would be undefined.
Use this instead.
```suggestion
        apiKey: BUGSNAG_API_KEY,
```